### PR TITLE
docs(Entry-Context): added short description

### PR DESCRIPTION
**Pull request by tarang9211**
added short description for the entry and context configurations

### CODE REVIEW

**Comment by tarang9211** [comment]
@bebraw & @TheLarkInn let me know if you think the description is good!

File: content/configuration/entry-context.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ sort: 4
 contributors:
   - sokra
   - skipjack
+  - tarang9211
 ---
 
-?> TODO: Short description
+?> The entry object is where Webpack looks to start building the bundle. The context is an absolute string to the directory that contains the entry files.
```
Comment (bebraw) [inline content/configuration/entry-context.md]: You can remove the `?>` prefix as that signifies a todo. Good otherwise.

**Comment by tarang9211** [comment]
I have signed it.

File: content/configuration/entry-context.md (hunk 1 above)
Comment (bebraw) [inline content/configuration/entry-context.md]: Can you change this too so the formatting is right? Thanks.

Comment (tarang9211) [inline content/configuration/entry-context.md]: I'm a little to this formatting, what exactly is incorrect with it?

Comment (bebraw) [inline content/configuration/entry-context.md]: Just `The entry object...` without the todo prefix.

Comment (tarang9211) [inline content/configuration/entry-context.md]: @bebraw I'm a little new to this kind of formatting. What exactly do you want me to change?

File: content/configuration/entry-context.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ contributors:
   - tarang9211
 ---
 
-?> The entry object is where Webpack looks to start building the bundle. The context is an absolute string to the directory that contains the entry files.
+The entry object is where Webpack looks to start building the bundle. The context is an absolute string to the directory that contains the entry files.
```
Comment (tarang9211) [inline content/configuration/entry-context.md]: This is what we're going for, right?

File: content/configuration/entry-context.md (hunk 1 above)
Comment (tarang9211) [inline content/configuration/entry-context.md]: Oh ok, have a look at the latest commit!

**Comment by tarang9211** [comment]
@bebraw im a little confused, although this has been merged into master, I don't see myself on the contributors page https://github.com/webpack/webpack.js.org

**Comment by bebraw** [comment]
@tarang9211 Where are you looking? See footer at https://webpack.js.org/configuration/entry-context/ .

screenshot 2017-01-29 12 09 41

**Comment by bebraw** [comment]
If you meant readme, that's aggregated based on https://opencollective.com/webpack donations.

**Comment by tarang9211** [comment]
@bebraw im looking here -> https://github.com/webpack/webpack.js.org/graphs/contributors

**Comment by bebraw** [comment]
I don't know about that. Why is this important?

**Comment by tarang9211** [comment]
I was curious. I made some contributions toward loader-runner and I show up there but not here.

**Comment by bebraw** [comment]
The commit definitely shows up at https://github.com/webpack/webpack.js.org/commits/master/content/configuration/entry-context.md, but I don't know why GitHub isn't picking that up.

**Comment by tarang9211** [comment]
yea, super weird. I have everything configured properly too. Oh well.

**Comment by bebraw** [comment]
I guess one option is that squashing the commits triggers the behavior. It went as one commit to the repo.

**Comment by tarang9211** [comment]
Yea, let's try not squashing the next PR. I'm just curious, but if that goes against your merging policies, I understand.

**Pull request merged**